app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/attachment/AttachmentOptions.kt

```diff
@@ -39,7 +41,8 @@ fun AttachmentOptionsComponent(
     onSendAttachment: (AttachmentBundle?) -> Unit,
     onError: (String) -> Unit
 ) {
-    val attachmentOptions = buildAttachmentOptionItems { pickedUri -> attachmentInnerState.pickAttachment(pickedUri) }
+    val scope = rememberCoroutineScope()
```

Comment:
if we scope the coroutinescoep to this Composable it will be recomposoed pretty often I think, Not sure if this is something you want but I think it is worth to test it by checking how it behaves when you close AttachmentOptionsComponent for example, how the cancellation of the job is going